resources/js/curl-to-go.js

```diff
@@ -193,7 +193,7 @@ function curlToGo(curl) {
 		else if (method == "HEAD")
 			return 'resp, err := http.Head('+goExpandEnv(url)+')\n'+err+deferClose;
 		else
-			return 'req, err := http.NewRequest('+goExpandEnv(method)+', '+goExpandEnv(url)+', nil)\n'+err+'resp, err := http.DefaultClient.Do(req)\n'+err+deferClose;
+			return 'req, err := http.NewRequest(http.Method'+toTitleCase(method)+', '+goExpandEnv(url)+', nil)\n'+err+'resp, err := http.DefaultClient.Do(req)\n'+err+deferClose;
```

Comment:
Thanks, looks pretty good, although we lost the `goExpandEnv()` here.

Reply:
That was only used to add the double quotes in this case, I'm assuming.

> // goExpandEnv adds surrounding quotes around s to make it a Go string,
> // escaping any characters as needed. It checks to see if s has an
> // environment variable in it. If so, it returns s wrapped in a Go
> // function that expands the environment variable. Otherwise, it
> // returns s wrapped in quotes and escaped for use in Go strings.
> // s should not already be escaped! This function always returns a Go
> // string value.

I'd be surprised if the environment part was relevant here 🤔

Reply:
Hm, okay, probably fine then. We can see if anyone was using that feature :smile: